src/homework/day8/util/UtilClassForMap.java

```diff
@@ -0,0 +1,38 @@
+package homework.day8.util;
+
+import java.util.List;
+import java.util.Map;
+
+import static homework.day8.collectionsforclasses.SandList.random;
+
+public class UtilClassForMap<T> {
+
+    public Map<Integer, T> fillMapWithObjects(List<T> anyList, Map<Integer, T> anyMap) {
+        for (T item : anyList) {
+            anyMap.put(random.nextInt(1, 10000), item);
```

Comment:
via randomizer value could be not unique, but it's not critical as for me, could leave as is

Reply:
fixed